internal/bminventory/inventory.go

```diff
@@ -1590,7 +1590,7 @@ func (b *bareMetalInventory) UpdateClusterInternal(ctx context.Context, params i
 	log := logutil.FromContext(ctx, b.log)
 	var cluster common.Cluster
 	var err error
-	log.Info("update cluster ", params.ClusterID)
+	log.Infof("update cluster %s with params: %+v", params.ClusterID, params)
```

Comment:
I'm assuming you wanted it to print the info under `ClusterUpdateParams`, but currently it does not display much (took it from artifacts svc log):

```golgng
time="2021-02-15T13:04:13Z" level=info msg="update cluster b7f31635-b575-41d8-9508-5f84ac6f67c9 with params: {HTTPRequest:0xc00123b800 ClusterUpdateParams:0xc00123b900 ClusterID:b7f31635-b575-41d8-9508-5f84ac6f67c9}" func="github.com/openshift/assisted-service/internal/bminventory.(*bareMetalInventory).UpdateClusterInternal" file="/go/src/github.com/openshift/origin/internal/bminventory/inventory.go:1593" cluster_id=b7f31635-b575-41d8-9508-5f84ac6f67c9 go-id=12137 pkg=Inventory request_id=0a65b347-c5a7-4063-bb83-dd4f3bcb70ed
```  
So probably we should handle the print of `ClusterUpdateParams` here.

Reply:
right